### ISSUE TITLE
right-click-mex cannot be cancelled if...

### ISSUE THREAD

**Issue by Rhade1** [beyond-all-reason/Beyond-All-Reason#698]
It is impossible to cancel a queued extractor, if another command (I only tested with Move) is given after the right-click-mex command. 
So
right-click-mex -then-> move. Then I was unable to cancel the right-click-mex command, without clearing the whole queue.
Other commands may cause this issue too.

**Comment by Rhade1** [comment]
Right-click mex sometimes cleared a construction queue for builders.

**Comment by Ruwetuin** [comment]
it is possible to cancel such queued extractor by placing/dragging an area mex command over it...

we would need to convert it to an actual sequence of regular buidl commands to "fix"
